authors/apps/authentication/views.py

```diff
@@ -25,8 +35,41 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
         serializer.save()
-
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        address = serializer.data['email']
+        user = User.objects.filter(email=user['email']).first()
+
+        RegistrationAPIView.generate_activation_link(user, request)
+        return Response({"message": "A verification email has been sent to {}".format(
+            address)}, status=status.HTTP_201_CREATED)
+
+    @staticmethod
+    def generate_activation_link(user, request=None, send=True):
```

Comment:
Please add **UNIT** test for this code. Remember to mock the `send_mail` call

Reply:
Could you please elaborate on that?
You want me to test the `generate_activation_link` function or ...🤔

Reply:
Yeah, create a test for only this function/logic. Mock out other stuff like external calls.

Reply:
@collin5 All the function does is return a token that I would use in tests for verifying a user. Maybe also sending the email but I had already tested that the email is sent.
So I have tested that the function does return a token.